subprojects/maven/src/main/java/org/gradle/api/publication/maven/internal/pom/DefaultMavenPom.java

```diff
@@ -133,6 +134,16 @@ public DefaultMavenPom project(Closure cl) {
         return this;
     }
 
+    @Override
+    public DefaultMavenPom project(final Action<? super GroovyObject> action) {
+        return project(new Closure(this, this) {
+            @SuppressWarnings("unused")
+            public void doCall() {
+                action.execute((GroovyObject) getDelegate());
+            }
+        });
+    }
```

Comment:
This one can't be because it's by design dynamically dispatched via Groovy. See https://github.com/gradle/gradle/blob/fdce92e976911e651f0066c6c8e2ce703836b634/subprojects/maven/src/main/java/org/gradle/api/publication/maven/internal/pom/DefaultMavenPom.java#L133